src/components/customMdx/docLink.tsx

```diff
@@ -16,7 +18,7 @@ const icons = {
   display: <Display />,
   code: <Code />,
 }
-const DocLinkWrapper = styled.a`
+const DocLinkWrapper = styled(Link)`
```

Comment:
unrelated changes in this file?
intentional?

Reply:
@janpio This change is needed to make sure the links point to one with the pathprefix. Gatsby links does this under the hood.